packages/react/src/components/Accordion/AccordionContent.tsx

```diff
@@ -20,6 +20,9 @@ export interface AccordionContentProps
   /** Whether or not the content is visible. */
   active?: boolean
 
+  /** Whether or not content should be indented */
+  indented?: boolean
```

Comment:
Can you please:
- explain why this prop is relevant in describing the content of an accordion? 
- describe the scenario where this prop is needed?

We need a strong argument with examples (preferably from the larger web community) that a prop is describing a component before we can accept it as a valid prop

@kuzhelov 
@layershifter 
@miroslavstastny 
@mnajdova 
@alinais

Reply:
So I definitely don't like the feeling of having this prop. It is an artifact of the way Accordion renders the Title and Content of each panel separately within the Accordion. The purpose of Indented is to react to the presence of the Icon (that is a question in itself) and indent the accordion content accordingly.
With and without Icon:
![image](https://user-images.githubusercontent.com/15043460/53195811-56a81500-35cb-11e9-8b44-6f019d1c9e1a.png)